README.md

```diff
@@ -106,7 +106,6 @@ Use the Azure cloud provider and filter on instances that have both tag key
 ```bash
 river-guide --provider azure \
             --tags "Environment=dev,DashboardManageable=true" \
-            --resource-group-name "my-resource-group" \
```

Comment:
Why did we get rid of this example? Seems like a good opportunity to explain how to filter with a resource group?

Reply:
Need to update the associated comment...